server/routes/flora.js

```diff
@@ -29,18 +29,12 @@ router.patch('/flora/:id', setUser, async (req, res, next) => {
   const update = { ...req.body }
   delete update._id
   const flora = await updateModel(Flora, req.params.id, update)
-  if (flora) return res.json(flora)
-  else {
-    res.status(400).send('Flora not updated/found')
-  }
+  flora ? res.json(flora) : res.status(400).send('Flora not updated/found')
```

Comment:
Split the error up if possible, will be easier to debug in future and also better error handling